packages/lib/src/account/Account.ts

```diff
@@ -65,12 +63,29 @@ export function getProfileCreationMessage(stringifiedProfile: string) {
     return `Hearby your dm3 profile is linked with your Ethereum account\n\n ${stringifiedProfile}`;
 }
 
+/**
+ * calculate the namehash of a given ENS name
+ * @param account Account with ENS name
+ */
+export function getNamehash(account: Account): string {
```

Comment:
Haven't we decided to not do nameprep anymore because it causes errors if the ens-name is too long?